src/components/Providers/helpers.js

```diff
@@ -0,0 +1,9 @@
+import { SETTINGS_DOCTYPE } from 'src/doctypes'
+
+export const saveAccountToSettings = async ({ client, setting, account }) => {
+  await client.save({
+    ...setting,
+    account,
+    _type: SETTINGS_DOCTYPE
+  })
+}
```

Comment:
```
export const saveAccountToSettings = ({ client, setting, account }) => 
  await client.save({
    ...setting,
    account,
    _type: SETTINGS_DOCTYPE
  })
```

Reply:
because,
1. it's better to return the result of the function
2. you can use arrow function
3. async/await are not relevant

Reply:
sans le await tu veux dire j'imagine